src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -265,12 +265,12 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 </PropertyGroup>
 
 <Choose>
-	<When Condition=" '$(DebugSymbols)' != '' And $(DebugSymbols) And '$(DebugType)' != '' And '$(Optimize)' != 'True' ">
+	<When Condition=" '$(DebugSymbols)' != '' And $(DebugSymbols) And '$(DebugType)' != '' And '$(DebugType)' != 'None' And '$(Optimize)' != 'True' ">
```

Comment:
Why is `$(DebugType)` required here? If `$(Optimize)` is False (!True), then we should treat the .apk as a Debug build...shouldn't we? Why do we care about `$(DebugType)` at all?